kit/database.go

```diff
@@ -78,6 +78,34 @@ func database(args []string) error {
 		}
 
 		fmt.Println("✅ Database dropped successfully")
+
+	case "reset":
+		err := db.Drop(url)
+		if err != nil {
+			return err
+		}
+
+		err = db.Create(url)
+		if err != nil {
+			return err
```

Comment:
@MateoCaicedoW can we wrap these errors so the message explains better what happened?

Reply:
sure!